lonestar/analytics/distributed/tc/tc.cpp

```diff
@@ -115,28 +164,25 @@ int main(int argc, char** argv) {
 #ifdef GALOIS_ENABLE_GPU
   std::tie(hg, syncSubstrate) =
       distGraphInitialization<NodeData, void>(&cuda_ctx, false);
-#else
-  std::tie(hg, syncSubstrate) = distGraphInitialization<NodeData, void>();
-#endif
-
   std::string timer_str("SortEdgesGPU");
   galois::StatTimer edgeSortTime("SortEdgesGPU", regionname);
   edgeSortTime.start();
   sort_cuda(cuda_ctx);
   edgeSortTime.stop();
+#else
+  std::tie(hg, syncSubstrate) = distGraphInitialization<NodeData, void>();
```

Comment:
Pass in "false" to distGraphInitialization. It makes it so proxy edge metadata isn't created for sync, which will take up more memory than necessary.